docs/development/oidc.md

```diff
@@ -0,0 +1,85 @@
+---
+title: OIDC
```

Comment:
How about something with a bit more?

> Testing OIDC with Kubernetes API Server: Step-by-Step Guide

Probably not that^... but something to indicate it's about testing OIDC and Headlamp.

Reply:
Updated, please review.